project/DependsOn.scala

```diff
@@ -0,0 +1,29 @@
+import sbt._
+
+object DependsOn {
+
+  def scalatest(scalaVersion: String): ModuleID = {
+    val artifact = "org.scalatest" %% "scalatest"
+    if (scalaVersion.startsWith("2.12")) artifact % "3.0.0" % Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "2.12" appears 4 times in the file.](https://www.codacy.com/app/pathikrit/better-files/file/3977902109/issues/source?bid=2860326&fileBranchId=3676352#l7)